pkg/checkpoint/checkpointvaliditychecker/conservativecvc.go

```diff
@@ -0,0 +1,64 @@
+package checkpointvaliditychecker
+
+import (
+	es "github.com/go-errors/errors"
+
+	"github.com/filecoin-project/mir/pkg/checkpoint"
+	"github.com/filecoin-project/mir/pkg/crypto"
+	issconfig "github.com/filecoin-project/mir/pkg/iss/config"
+	checkpointpbtypes "github.com/filecoin-project/mir/pkg/pb/checkpointpb/types"
+	trantorpbtypes "github.com/filecoin-project/mir/pkg/pb/trantorpb/types"
+	tt "github.com/filecoin-project/mir/pkg/trantor/types"
+	t "github.com/filecoin-project/mir/pkg/types"
+)
+
+type ConservativeCVC struct {
+	issParams    *issconfig.ModuleParams
+	ownID        t.NodeID
+	hashImpl     crypto.HashImpl
+	chkpVerifier checkpoint.Verifier
+}
+
+// NewConservativeCVC returns a new ConservativeCVC. This checkpoint validity checker simply rejects checkpoints whose signatures cannot be verified because
+// the node does not know the membership of the relevant epoch yet.
```

Comment:
This makes total sense, and I'd add another CVC that does the same thing, but allows new checkpoints with a huge warning, as we do currently.

Reply:
PermissiveCVC, done (coming submission)